{{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml

```diff
@@ -66,10 +66,11 @@ jobs:
         environment-file: devtools/conda-envs/test_env.yaml
         add-pip-as-python-dependency: true
 {% if cookiecutter.__dependency_source == 'conda-forge' %}
-        miniforge-variant: Mambaforge
+        miniforge-version: "latest"
         use-mamba: true
         channels: conda-forge, defaults
```

Comment:
Same things here re: defaults

Reply:
Done, and raised #142 to tackle environment files.